app/controllers/api/verify/base_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Api
   module Verify
     class BaseController < ApplicationController
+      skip_before_action :verify_authenticity_token
```

Comment:
do we want to skip the authenticity token check?

Reply:
Yes because its an API call.